README.md

```diff
@@ -55,18 +48,8 @@ Your app is ready to be deployed!
 
 See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
 
-### `npm run eject`
```

Comment:
I think we can save this block in readme. Any specific reason for removing it?

Reply:
I find `npm run eject` an advanced command and thought the one could read about it in the documentation of create-react-app which is linked at the end of README. 

I thought this would help with keeping the README concise. What are you thoughts regarding this?